jbehave-core/src/main/java/org/jbehave/core/steps/StepCreator.java

```diff
@@ -731,17 +730,18 @@ public String asString(Keywords keywords) {
         }
     }
 
-    private class BeforeOrAfterStep extends AbstractStep {
+    private class BeforeOrAfterStep extends ReportingAbstractStep {
```

Comment:
just to confirm: do we still need both `AbstractStep` and `ReportingAbstractStep`?

Reply:
decided to left it as it is to keep unchanged the string values returned for reporting steps and Upon(Any|Failed|Success)Step that use default implementation